frontend/apps/monkvision/components/chart-box/chart-box.html

```diff
@@ -163,6 +164,12 @@
     </table>
     {{/table}}
 
+    {{#chartMenuEnabled}}           <!-- Handles menu on chart -->
+        <span-with-menu id="btn_{{elementId}}" styleBody='div.menuitem{color: rgba(80,80,80,0.9);cursor: pointer;} div.menuitem:hover{background: rgba(80,80,80,0.9) !important; color: rgba(250,250,250,0.95) !important;} span#content > i:hover{filter: brightness(70%);transition: all 1s ease;} div#menu{margin-left: -131px;border-radius: 8px 0px 8px 8px;margin-top:10px;}' style="float:right; padding-right:10px;">
+            <i class="fa fa-ellipsis-v fa-2x" aria-hidden="true" data-value={{elementId}} style='position:absolute; cursor: pointer;'></i>
```

Comment:
We should not have to include an entire library (`font-awesome`) just to use a single icon.
Suggesting adding the icon to `monkvision/frontend/apps/monkvision/img/` as `.png` or `.svg` to support both dark and light themes.

Reply:
We should not use font-awesome at all. Use SVG as needed or UNICODE.

Reply:
Thanks for catching this - Sridharan - please DO NOT EVER use any external library - Javascript, CSS etc. without explicit approvals. We always strive to NOT use external libraries, as far as possible.